public/locales/en-US/translations.json

```diff
@@ -392,11 +378,12 @@
   "uri": "URI",
   "owner": "Owner",
   "other_chain_destination": "Other Chain Destination",
-  "%_of_total_validators":"% of Total Validators",
-  "version_display":"Version: {{version}}",
-  "validators_count":"# of Validators: {{count}}",
+  "%_of_total_validators": "% of Total Validators",
+  "version_display": "Version: {{version}}",
+  "validators_count": "# of Validators: {{count}}",
   "current_stable_version": "Current Stable Version",
   "stable_version": "{{stableVersion}}",
   "xchain_claim_attestation": "Claim Attestation",
-  "xchain_create_account_attestation": "Create Account Attestation"
+  "xchain_create_account_attestation": "Create Account Attestation",
+  "transaction_AccountDelete": "Account Delete"
```

Comment:
None of the other names have capitals, and they use underscores for spaces.
```suggestion
  "transaction_account_delete": "Account Delete"
```

Reply:
<img width="631" alt="Screen Shot 2022-09-29 at 12 38 19 PM" src="https://user-images.githubusercontent.com/9340787/193126207-6b6609ef-c10f-451c-af18-d661b6da7aa6.png">
all the other transaction related have capitals

Reply:
@JST5000 the thing you are looking at the string used to describe a collection of attestations which isnt the actual transaction type.

Reply:
Ah, nevermind then :)